Documentation/AEPMedia.md

```diff
@@ -0,0 +1,1191 @@
+[TOC]
+
+# Getting Started
+This section walks through how to get up and running with the AEP Swift Media SDK with only a few lines of code.
+
+## Set up a Mobile Property
+Set up a mobile property as outlined in the Adobe Experience Platform [docs](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property)
+
+## Get the Swift Mobile Media
+
+Now that a Mobile Property is created, head over to the [install instructions](https://github.com/adobe/aepsdk-media-ios#installation) to install the SDK.
+
+## Initial SDK Setup
+
+**Swift**
+
+1. Import each of the core extensions in the `AppDelegate` file:
+
+```swift
+import AEPCore
+import AEPMedia
+import AEPAnalytics
+import AEPIdentity
+```
+2. Register the core extensions and configure the SDK with the assigned application identifier.
+To do this, add the following code to the Application Delegate's `application(_:didFinishLaunchingWithOptions:)` method:
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+ // Enable debug logging
+ MobileCore.setLogLevel(level: .trace)
+
+ MobileCore.registerExtensions([Media.self, Identity.self, Analytics.self], {
+ // Use the App id assigned to this application via Adobe Launch
+ MobileCore.configureWith(appId: "appId") 
+ })  
+ return true
+}
+```
+**Objective C**
+
+1. Import each of the core extensions in the `AppDelegate` file:
+
+```objective-c
+@import AEPCore;
+@import AEPMedia;
+@import AEPAnalytics;
+@import AEPIdentity;
+```
+2. Register the core extensions and configure the SDK with the assigned application identifier.
+    To do this, add the following code to the Application Delegate's 
+    `application didFinishLaunchingWithOptions:` method:
+
+```objective-c
+(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+
+  // Enable debug logging
+  [AEPMobileCore setLogLevel: AEPLogLevelTrace];
+    
+  [AEPMobileCore registerExtensions:@[AEPMobileIdentity.class, AEPMobileLifecycle.class, AEPMobileAnalytics.class, AEPMobileMedia.class] completion:^{
+  // Use the App id assigned to this application via Adobe Launch
+  [AEPMobileCore configureWithAppId:@"appId"];
+   }];
+   return YES;
+}
+```
+
+
+
+
+# Media API reference
+This section details all the APIs provided by Media Analytics, along with sample code snippets on how to properly use the APIs.
+
+## createTracker
+Creates a media tracker instance that tracks the playback session. The tracker created should be used to track the streaming content, and it sends periodic pings to the media analytics backend.
+
+{% hint style="info" %}
```

Comment:
Hint tags are not supported in markdown. Can we just use an emoji like (⚠️ ) to show it is an hint